Source/OxyPlot/Series/BarSeries/LinearBarSeries.cs

```diff
@@ -195,7 +195,14 @@ protected internal override void UpdateAxisMaxMin()
         {
             base.UpdateAxisMaxMin();
 
-            this.YAxis.Include(0.0);
+            if (this.YAxis.IsLogarithmic())
+            {
+                this.YAxis.Include(double.Epsilon);
```

Comment:
Not sure this is desirable for `LinearBarSeries`: on a log axis, only the different between the bars matters, so we don't want to see the baseline if it's not set explicitly.